### PR TITLE
feat: add some fzf-lua keymap

### PR DESCRIPTION
**Pull request by iamxiaojianzheng**
- `ctrl-s` search_in_project_files
- `ctrl-w` change_working_directory
- `ctrl-d` delete_project
- `ctrl-r` recent_project_files

### CODE REVIEW

**Comment by iamxiaojianzheng** [comment]
There may be a better way to implement these features, but I've done my best.

**Comment by dpetka2001** [comment]
Maybe you'd also like considering changing the current `<ctrl-t>` into 
```lua
        ["ctrl-t"] = {
          function(selected)
            vim.cmd("tabedit")
            fzf_lua.files({ cwd = selected[1] })
          end,
        },
```
I find this one better than openening in a new tab the directory in Netrw file manager. Try it out to see if you like it.

**Comment by iamxiaojianzheng** [comment]
> Maybe you'd also like considering changing the current `<ctrl-t>` into也许您还想考虑将电流 `<ctrl-t>` 更改为
> 
> ```lua
>         ["ctrl-t"] = {
>           function(selected)
>             vim.cmd("tabedit")
>             fzf_lua.files({ cwd = selected[1] })
>           end,
>         },
> ```
> 
> I find this one better than openening in a new tab the directory in Netrw file manager. Try it out to see if you like it.我发现这比在新选项卡中打开 Netrw 文件管理器中的目录要好。试试看你是否喜欢它。

emmm, `ctrl-t` was not added by me.

@folke What do you think of the comments made by dpetka2001?

**Comment by folke** [comment]
That's fine. I dont use project.nvim, so all good to me :)

**Comment by dpetka2001** [comment]
Me neither. I just think this one is better suited to the person in the Discussion that asked about this.

File: lua/lazyvim/plugins/extras/util/project.lua [hunk 1]
```diff
@@ -1,11 +1,15 @@
-local pick = function()
+local pick = nil
+
+pick = function()
```
Comment (dotfrag) [inline lua/lazyvim/plugins/extras/util/project.lua:3]: Why this change?

File: lua/lazyvim/plugins/extras/util/project.lua [hunk 2]
```diff
@@ -14,6 +18,34 @@ local pick = function()
...
+            end
+          end,
+        },
+        ["ctrl-d"] = function(selected)
+          local path = selected[1]
+          local choice = vim.fn.confirm("Delete '" .. path .. "' project? ", "&Yes\n&No")
+          if choice == 1 then
+            history.delete_project(path)
+          end
+          pick()
```
Comment (iamxiaojianzheng) [inline lua/lazyvim/plugins/extras/util/project.lua:47]: @dpetka2001 Used here

**Comment by folke** [comment]
Thanks!
